draft-rwbr-tsvwg-signaling-use-cases.md

```diff
@@ -278,6 +284,11 @@ connection establishment, so there needs to be a way for the client
 to authorize the network elements to receive and hopefully to honor the metadata of those
 packets.
 
+Without a separate signaling protocol, remote peers are able to mark every packet of a flow
```

Comment:
Idem as above. 

"separate" is not important here as far as the signal is provided.

The value of having "separate" is to factorize the same signaling independent of the application, transport protocol, etc.